hmpps-prisoner-search-indexer/src/main/kotlin/uk/gov/justice/digital/hmpps/prisonersearch/indexer/services/IndexQueueService.kt

```diff
@@ -32,25 +35,27 @@ class IndexQueueService(
   private val indexDlqUrl by lazy { indexQueue.dlqUrl as String }
 
   fun sendIndexMessage(index: SyncIndex, type: IndexRequestType) {
-    val result = indexSqsClient.sendMessage(
-      SendMessageRequest.builder().queueUrl(indexQueueUrl).messageBody(
-        objectMapper.writeValueAsString(
-          IndexMessageRequest(type = type, index = index),
-        ),
-      ).build(),
-    ).get()
-    log.info("Sent populate index message request {}", result.messageId())
+    sendMessage(IndexMessageRequest(type = POPULATE_INDEX, index = index)).also {
```

Comment:
was a bit naff to pass through the `IndexRequestType` here, so will change to have 6 methods, one for each message sent